rfc6962-bis.xml

```diff
@@ -424,6 +424,9 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
             </t>
           </list>
         </t>
+        <t>
+          Note that this mechanism allows redaction of any type of field in the subjectAltName extension.
```

Comment:
"redaction of any type of field" is misleading.  Section 4.3 is specific about the conditions under which end-entity certs may be excused from having to be logged.  If a CA doesn't meet those conditions (relating to dNSNames and IP Addresses) then they can't expect to "redact" a directoryName, SRVName, etc, even those these qualify as "any type of field".

BTW, I think we should only use the term "redaction" to refer to the section 4.2 mechanism.

If you make the change I suggested in my other comment, I think this new paragraph is probably redundant.